packages/ethereum/contracts/src/utils/EnumerableTargetSet.sol

```diff
@@ -0,0 +1,158 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (last updated v4.8.0) (utils/structs/EnumerableSet.sol)
+// This file was procedurally generated from scripts/generate/templates/EnumerableSet.js.
+
+pragma solidity ^0.8.0;
+
+import "./TargetUtils.sol";
+
+struct TargetSet {
```

Comment:
I do not understand what is going on here - this struct needs a better explanation, e.g. what are keys and what are values in the mapping? Why does this need to be tied together in a struct?